kibana-reports/server/backend/opendistro-reports-scheduler-plugin.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { REPORTS_SCHEDULER_API } from '../../common';
+
+export default function (Client: any, config: any, components: any) {
+  const ca = components.clientAction.factory;
```

Comment:
Could you use a more descriptive variable name here?

Reply:
use `clientAction` instead